charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "zookeeper-operator.commonLabels" . | indent 4 }}
+{{ toYaml .Values.labels | indent 4 }}
```

Comment:
Could you add the labels to this section instead?
https://github.com/pravega/zookeeper-operator/blob/fef0547b86adab72e0700e7303531f17d30bedd8/charts/zookeeper-operator/templates/operator.yaml#L16-L18